ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```
@@ -0,0 +1,785 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  ethernet_interfaces:
+    type: list
+    primary_key: name
+    convert_types:
+    - dict
+    description: Ethernet interfaces
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+        peer:
+          type: str
+        peer_interface:
+          type: str
+        peer_type:
+          type: str
+        description:
+          type: str
+        shutdown:
+          type: bool
+        speed:
+          type: str
+          description: Speed can be interface_speed or forced interface_speed or auto interface_speed
+        mtu:
+          type: int
+          convert_types:
+          - str
+        l2_mtu:
+          type: int
+          convert_types:
+          - str
+          description: If l2_mtu defined this profile should only be used for platforms supporting the "l2 mtu" CLI
```

Using multiline to ensure proper quoting.
```suggestion
          description: |
            "l2_mtu" should only be defined for platforms supporting the "l2 mtu" CLI
```